uvicorn/protocols/websockets/auto.py

```diff
@@ -1,12 +1,13 @@
 import asyncio
```

Comment:
On the tests is fine, but can we revert it here? I guess the cost is higher to import importlib + call import_module than import the packages itself?

Reply:
Yeah, I guess it's not worth it here.